src/views/preferences/list/ClusterPreferenceList.scss

```diff
@@ -0,0 +1,9 @@
+.cluster-preference-list {
```

Comment:
this is identical to cluster-instancetype-list - maybe just use one of them? with a more appropriate name?

Reply:
we this code repeating in few more places, any sense in util/scss ?